areas/extension/src/AzureMcp.Extension/Commands/AzCommand.cs

```diff
@@ -1,47 +1,24 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
Does this PR mean MCP doesn't execute the commands and tells the user how to execute them?